packages/atlas/src/components/_comments/Comment/Comment.tsx

```diff
@@ -239,6 +262,39 @@ export const Comment: React.FC<CommentProps> = ({
                     {!allReactionsApplied && !isDeleted && (
                       <ReactionPopover disabled={reactionIsProcessing} onReactionClick={handleCommentReactionClick} />
                     )}
+                    <RepliesWrapper>
+                      {!!repliesCount && (
+                        <StyledAvatarGroup
+                          size="small"
+                          avatars={
+                            replyAvatars?.filter((avatar, idx) => !mappedAvatars?.includes(avatar.url, idx + 1)) ||
```

Comment:
This will not work if two users have the same avatar. 
Also, I think we could do this a little better. There is a nice trick that will remove duplicates. You could do this at line 98
```js
  const avatarsWithoutDuplicates = [...new Map(replyAvatars?.map((item) => [item.handle, item])).values()]
```